posts/epiparameter_v0.1.0/index.qmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "epiparameter v0.1.0"
+author:
+  - name: "The Epiverse-TRACE development team"
+date: "2024-05-03"
```

Comment:
Should be updated before merging.

Reply:
Date updated in ec67d8e966aac1cc5c11be3f686138a68037e57f.